soc/xtensa/intel_s1000/dts_fixup.h

```diff
@@ -12,8 +12,8 @@
 #define DT_UART_NS16550_PORT_0_IRQ_FLAGS	DT_NS16550_80800_IRQ_0_SENSE
 #define DT_UART_NS16550_PORT_0_CLK_FREQ	DT_NS16550_80800_CLOCK_FREQUENCY
 
-#define DT_L2_SRAM_BASE				DT_SRAM_BASE_ADDRESS
-#define DT_L2_SRAM_SIZE				DT_SRAM_SIZE * 1024
+#define DT_L2_SRAM_BASE				CONFIG_SRAM_BASE_ADDRESS
+#define DT_L2_SRAM_SIZE				CONFIG_SRAM_SIZE * 1024
```

Comment:
Personally, I don't like using CONFIG_* macros in `dts_fixup.h` files. It feels strange.